projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-ai.php

```diff
@@ -117,6 +119,137 @@ public function register_basic_routes() {
 				),
 			)
 		);
+
+		register_rest_route(
```

Comment:
one thing with proxying this endpoint is that AFAIK we support streaming on WPCOM end and I don't think every server running JP does, so this can break streaming

Reply:
Interesting. What are the server requirements for streaming to work?

Reply:
The server has to support Server-Side Events